src/geometry.rs

```diff
@@ -859,7 +919,19 @@ impl Plane {
     }
 
     pub fn normal(&self) -> Vector3<f32> {
```

Comment:
In case of `normal()` you return a `Vector3` and in case of `x_vector`, `y_vector` and `origin` you return a `&Vector3`. Let's be consistent. It is not super clear what the correct answer is, let's pick the owned version.

Reply:
because the referenced ones are just extractions of some existing value, the normal is calculated (created anew). nevertheless, i'm doing as you say